src/test/java/Homework21.java

```diff
@@ -0,0 +1,56 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.Test;
+
+public class Homework21 extends BaseTest{
+    String newPlayListName = "NewPlayListName";
+    String currentPlayListName = "HW17";
+    @Test
+    public void renamePlaylist() throws InterruptedException {
+
+        login("dmytro.kravchuk@testpro.io", "Fr440003");
+        WebElement playList =  wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath(String.format("//section[@id='playlists']//li/a[text()='%s']", currentPlayListName))));
+Thread.sleep(3000);
+        actions.contextClick(playList).perform();
+        WebElement editButton = wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath("//li[contains(@data-testid, 'playlist-context-menu-edit')]")));
+editButton.click();
+WebElement inputPlayListName = wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath("//ul/li//input[@type='text']")));
+for (int i=0; i>currentPlayListName.length(); i++) {
+    inputPlayListName.sendKeys(Keys.BACK_SPACE);
+}
```

Comment:
How can I beautify code in IntelliJ:

The formatting shortcuts in Intellij IDEA are :
For Windows : Ctrl + Alt + L.
For Ubuntu : Ctrl + Alt + Windows + L.
For Mac : ⌥ (Option) + ⌘ (Command) + L.

![image](https://user-images.githubusercontent.com/104761482/209774134-9e63bfed-d768-4455-a860-028305c85e17.png)